ee/codegen/src/__test__/nodes/code-execution-node.test.ts

```diff
@@ -93,7 +102,13 @@ describe("CodeExecutionNode", () => {
         });
 
         node.getNodeFile().write(writer);
-        expect(await writer.toStringFormatted()).toMatchSnapshot();
+        const output = await (override === "INLINE"
+          ? // Fern will error out when using toStringFormatted with our approach to inline code nodes
+            // with a bizarre syntax error, there's a try catch in our persisted file code and we also
+            // pass in a skipFormatting flag to it for this case
+            writer.toString()
+          : writer.toStringFormatted());
```

Comment:
[nit] this feels like we are letting implementation details bleed into tests. is it possible to do `node.persist()` to some temp file and then use toMatchSnapshot on the contents?